### PR TITLE
Iss13

### PR DESCRIPTION
**Pull request by LPMendoza**
# Description
The incidents and symptoms table were created to provide the mobile application a list of symptoms and rotes for register incidents.
 
  ## Screenshots 
image

### CODE REVIEW

**Review by jnegrete15 — DISMISSED** [review]

**Review by MiguelOcegueraM — APPROVED** [review]

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -1,13 +1,21 @@
 # frozen_string_literal: true
 class SessionsController < Devise::SessionsController  
     skip_before_action :verify_authenticity_token
-    
     respond_to :json
     private
     def respond_with(resource, _opts = {})
-        render json: resource
+        resJson = {
```
Comment (OchoaHub) [inline app/controllers/sessions_controller.rb:7]: Try to use better names of variables;)
Res = Carne de vaca

File: test/models/incident_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (OchoaHub) [inline test/models/incident_test.rb:1]: If you don't need test, delete them

**Review by OchoaHub — APPROVED** [review]
Looks good, just be careful with that user and password on the DB

File: app/controllers/sessions_controller.rb (hunk 1 above)
Comment (MiguelOcegueraM) [reply app/controllers/sessions_controller.rb:7]: thank you bro

**Review by jnegrete15 — APPROVED** [review]

File: app/controllers/api/session_controller.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        }
+        render json: resJson
+    end
+    def respond_to_on_destroy
+        render json: {
+            "error": false,
+            "code": "SSN_CLOSE"
+        }
+    end
+end
```
Comment (LDVera) [inline app/controllers/api/session_controller.rb:22]: add a blank line in the end

**Review by LDVera — APPROVED** [review]
all looks good continue working!

**Review by MiguelOcegueraM — APPROVED** [review]

**Pull request merged**